pkg/ebpf/processor_funcs.go

```diff
@@ -406,5 +410,14 @@ func (t *Tracee) processSharedObjectLoaded(event *trace.Event) error {
 		return nil
 	}
 
-	return t.addHashArg(event, filePath, int64(fileCtime))
+	if t.config.Output.CalcHashes {
+		fileId := &fileUniqueId{}
+		if err := fileId.Stat(filePath); err != nil {
```

Comment:
Again, you can just use the inode and device from the event instead of probing the file.